src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/constants/TSEConstants.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.constants;
+
+public final class TSEConstants {
+
+    public static final String GIVE_DETAIL_MISSING = "Please upload a document or provide details in the text box.";
+    public static final String CLAIMANT_TSE_WITHDRAW_CLAIM = "Withdraw all or part of claim";
+    public static final String CLAIMANT_REP_TITLE = "Claimant Representative";
+    public static final String MISSING_DETAILS = "Please upload a document or provide details in the text box.";
```

Comment:
Is the MISSING_DETAILS different from GIVE_DETAIL_MISSING as they have identical content?

Reply:
resolved in [636cd8d](https://github.com/hmcts/et-ccd-callbacks/pull/2036/commits/636cd8db6d10aada24309e716e80960e2c3c561e)